app/views/CreateDiscussionView/index.tsx

```diff
@@ -149,35 +138,39 @@ class CreateChannelView extends React.Component<ICreateChannelViewProps, ICreate
 				contentContainerStyle={styles.container}
 				keyboardVerticalOffset={128}>
 				<StatusBar />
-				<SafeAreaView testID='create-discussion-view' style={styles.container}>
+				<SafeAreaView testID='create-discussion-view'>
 					<ScrollView {...scrollPersistTaps}>
-						<Text style={[styles.description, { color: themes[theme].fontSecondaryInfo }]}>{I18n.t('Discussion_Desc')}</Text>
-						<SelectChannel
-							server={server}
-							userId={user.id}
-							token={user.token}
-							initial={this.channel && { text: getRoomTitle(this.channel) }}
-							onChannelSelect={this.selectChannel}
-							blockUnauthenticatedAccess={blockUnauthenticatedAccess}
-							serverVersion={serverVersion}
-						/>
-						<FormTextInput
-							label={I18n.t('Discussion_name')}
-							testID='multi-select-discussion-name'
-							placeholder={I18n.t('A_meaningful_name_for_the_discussion_room')}
```

Comment:
Now we don't have placeholders anymore, we need to remove them from i18n files.

Reply:
removed.